packages/cozy-notifications/src/style.css

```diff
@@ -0,0 +1,102 @@
+h1, h2, h3, h4, h5, h6 {
+  display:block;
+  margin:0;
+  padding:0;
+}
+
+.tableRow {
+  border-bottom: 1px #d6d8da solid;
+  padding: 8px 0;
+  line-height: 1.5;
+  letter-spacing: 0.25px;
+}
+
+.tableRow {
+  border-bottom: 0;
+}
+
+/* header */
+.header__title a {
+  text-decoration: none;
+  color: inherit;
+  white-space: nowrap;
+}
+
+.header__desc {
+  white-space: nowrap;
+  text-align: right;
+}
+.header__desc__title {
+  color: #32363f;
+  font-weight: bold;
+  font-size: 16px;
```

Comment:
should it be in rem in order to be responsive?

Reply:
Em is not well supported by email-client, I could not find anything about Rem but since it is more recent, I would expect to have even less support. In emails, it is better to stick to the tried and true `px`.